samples/README.md

```diff
@@ -0,0 +1,41 @@
+# Samples compatibility with different FTD versions
+
```

Comment:
I would add a brief explanation why some samples do not work with certain versions. E.g., something like that:

```
As FTD API evolves, it changes between releases and does not always guarantee backward compatibility. Thus, some playbook samples might not work with previous versions of FTD.

We aim to keep playbooks up-to-date and make them all work with the latest FTD version. To check if a certain playbook is compatible with a previous FTD release, see the compatibility table below.
```